backend/api/Areas/Property/Mapping/Parcel/ParcelMap.cs

```diff
@@ -67,6 +67,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.Id, src => src.Id)
                 .Map(dest => dest.PID, src => ParcelConverter.ConvertPID(src.PID))
                 .Map(dest => dest.PIN, src => src.PIN)
+                .Map(dest => dest.PropertyTypeId, src => src.PropertyTypeId)
```

Comment:
This is for an edge case - it is currently possible to create a subdivision with no parent by creating a subdivision and then deleting the parent. In that case, this logic will ensure that a subdivision is not converted to a parcel during an update just because it doesn't have a parent parcel.